scipp-developer-minimal.yml

```diff
@@ -18,8 +18,8 @@ dependencies:
   - appdirs
   - ipympl
   - ipywidgets
-  - matplotlib-base=3.2.2 # version 3.3.3 causes issues with nodejs in JupyterLab
-  - numpy >=1.15.3
+  - matplotlib-base
+  - numpy
```

Comment:
The `>=1.15` should probably also be kept here if you are keeping it above?

Reply:
Thanks, I've put this back